app/js/lib/validation_dispatchers/base.js

```diff
@@ -36,13 +37,31 @@ var _ = require( 'underscore' ),
 				return;
 			}
 
+			if ( !this.fieldsRequiredForDispatchAreSet( selectedValues ) ) {
+				return;
+			}
+
 			this.previousFieldValues = selectedValues;
 
 			if ( this.beginActionCreationFunction ) {
 				store.dispatch( this.beginActionCreationFunction( selectedValues ) );
 			}
 			validationResult = this.validationFunction( selectedValues );
 			return store.dispatch( this.finishActionCreationFunction( validationResult ) );
+		},
+
+		fieldsRequiredForDispatchAreSet: function ( formValues ) {
+			var allFieldsSet = true;
+
+			if ( _.isObject( formValues ) && this.fieldValuesIfNotSet !== null ) {
+				_.each( this.fieldValuesIfNotSet, function ( initalValue, fieldName ) {
+					if ( initalValue === formValues[ fieldName ] ) {
```

Comment:
Won't this break when initial values are set by a banner and are not in need of change?